wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/FileContentProvider.java

```diff
@@ -18,14 +18,25 @@
 import org.eclipse.che.api.workspace.server.devfile.exception.DevfileException;
 
 /**
- * Some types of {@link org.eclipse.che.api.core.model.workspace.devfile.Component} may have
- * configuration located in a separate file. It fetches content of file by its name.
+ * Interface for fetching devfile-related content from repositories or raw URL-s. Used for
+ * retrieving devfile content as well as referenced files.
  *
  * @author Max Shaposhnyk
  * @author Sergii Leshchenko
  */
 public interface FileContentProvider {
 
+  /**
```

Comment:
is there any change or is method just moved up ?

Reply:
just reorder